slsim/lensed_system_base.py

```diff
@@ -1,22 +1,26 @@
 from abc import ABC, abstractmethod
 
+
 class LensedSystemBase(ABC):
     """Abstract Base class to create a lens system with all lensing properties required
     to render populations."""
 
-    def __init__(self, source_class, deflector_class):
+    def __init__(self, source_class, deflector_class, los_class):
```

Comment:
Input los_class is not used. Do you think we should define a variable using this input?